### DISCUSSION TITLE
Set relational key for O2M item in Create mode

### DISCUSSION THREAD

**Discussion by jsx101** [directus/directus#21784] [Feature Requests]
Currently on Directus v9 but I couldn't find anything about this feature so I believe it's not implemented yet.

For context, I have collection_a that has an O2M relationship with collection_b. 

I have a custom interface in collection_b that requires the value of the relational key (which is the id of the collection_a item), but when trying to create a collection_b item in collection_a form, that relational key is not set until I save the collection_a item too.

Would be nice if I can configure the relational key to be set from the start when creating the collection_b item.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
